engine/src/test/resources/testAppPackage/mydtapp/src/main/java/com/example/mydtapp/Application.java

```diff
@@ -20,6 +20,7 @@
 
 import org.apache.hadoop.conf.Configuration;
 
+import com.datatorrent.api.Module;
```

Comment:
@vrozov  I check import order in apex style jar , this order seems correct. org > org.apache > com> com.datatorrent